dts/arm/st/f7/stm32f723.dtsi

```diff
@@ -16,7 +16,7 @@
 	};
 
 	dtcm: memory@20000000 {
-		compatible = "st,stm32-dtcm";
+		compatible = "arm,dtcm";
```

Comment:
Also cc: @MaureenHelm (and maybe @agansari) re: if it would make sense to switch iMX to generic arm,dtcm either.

Reply:
@pfalcon well, i.MX RT does not use 'standard' TCM memories, but dts should be the same. Let's see @MaureenHelm's feedback.